content/paths/sign_requests__post_sign_requests.yml

```diff
@@ -0,0 +1,36 @@
+---
+operationId: post_sign_requests
+
+summary: Create Sign Request
```

Comment:
It seems to me that the sign request could be in 2 parts. A template, plus the actual request. The template(s) would be their own object and contain email subject, message, reminder message, etc. so if an admin makes a change to the template then ALL clients would get the changes and you wouldn't have to chase down people on older versions so they stop sending email messages with last month's slogan or typo. Plus, you get the benefits of monitoring changes to the email messages by the admins and not letting field personnel send out whatever they like (Assuming you can restrict lusers from creating their own templates).

Reply:
Currently for these APIs we're exposing the existing Sign Request APIs (https://signrequest.com/api/v1/docs/#tag/signrequest-quick-create) and "boxifying" them. They do have a concept of templates (in a different way, it's more of an already prepared document for signing) that will be a feature that Box releases post GA. Providing a way for admins to use templates for/restrict the email notifications could definitely be a future feature, but would not be something we could add for GA